active/003-seq-feature.md

```diff
@@ -0,0 +1,108 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
+---
+
+# Summary
+
+Introduce an `interval_metadata` object to allow for the storage and querying of intervals.
```

Comment:
Your friending naminator:

* `features` 
* `feature_intervals`
* `feaint`
* `feat.meta_d` (because rap)
* `interval_features` (benefit is that a SNP is arguably a non-interval feature)